src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,79 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
+  text-align: center;
+  width: 100%;
+}
+
+.w-subscribe__border {
+  border: 2px solid #f1f3f4;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  width: 100%;
+}
+
+.w-subscribe__button {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.w-subscribe__error {
+  color: $WARNING_COLOR;
+}
+
+.w-subscribe__field {
+  display: flex;
+  justify-content: space-between;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 8px;
+  text-align: left;
+}
+
+.w-subscribe__form {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 720px;
+  width: calc(100% - 32px);
+}
+
+.w-subscribe__input {
+  border: 2px solid #f1f3f4;
+  border-top-left-radius: 2px;
+  border-top-right-radius: 2px;
+  font: inherit;
+  font-size: 16px;
+  line-height: 20px;
+  padding: 8px;
```

Comment:
The design seems to call for more padding than this.

Reply:
I won't be able to apply padding to the inner text of the select element though, so it'll look like they're not lined up properly.